Remotion/Web/ClientScript/Scripts/SmartPage.ts

```diff
@@ -821,14 +816,6 @@ class SmartPage_Context
     this.ExecuteEventHandlers('onscroll');
   };
 
-  // Event handler for Window.OnResize.
-  public OnResize(): void
-  {
-    if (this._statusMessageWindow != null)
-      this.AlignStatusMessage(this._statusMessageWindow);
```

Comment:
We need to check if the status message is still getting aligned correctly on resize. If no we should try and fix it using css so that we can remove the code here.

Reply:
I tested this by doing a postback and then clicking the go back button, which triggers a message. I did this before the change and after the change. In both cases, the message is getting aligned correctly on resize.
@MichaelKetting Is there another issue that we have missed?

Reply:
did you test in both NovaViso theme and NovaGray theme?

You can also trigger it via `SmartPage_Context.Instance.ShowStatusIsSubmittingMessage()` in the browser's Javascript console.

Reply:
Yes, it works for both themes!